tests/test_app/test_composable.py

```diff
@@ -303,7 +303,7 @@ def main(self, val: int) -> int:
 
 def test_as_completed():
     """correctly applies iteratively"""
-    dstore = io_app.get_data_store(DATA_DIR, suffix="fasta", limit=3)
+    dstore = io_app_new.open_data_store(DATA_DIR, suffix="fasta", limit=3)
```

Comment:
same issue with `io_app_new` here, except since only the `open_data_store` function is being used, you may as well use it via the new top-level import (i.e. `from cogent3 import pen_data_store`)